### DISCUSSION TITLE
Any example to configure telescope-live-grep-args.nvim plugin

### DISCUSSION THREAD

**Discussion by martindavid** [LunarVim/LunarVim#3753] [Q&A] [answered]
I've been using lvim for several weeks now and love the experience so far. It has everything that I need (almost) except the full text search capability. I try to figure out how to make it work and find this telescope [extension](https://github.com/nvim-telescope/telescope-live-grep-args.nvim). But I can't figure out how to configure it along with the existing lvim setup? Anyone has experience with that extension?

**Comment by LostInTheLogs** [comment, marked as answer]
you can live grep by default in lvim, it's mapped to `leader s t`, that extension only adds the ability to pass arguments to rg

**Reply by serguei-at-bemaximus** [reply]
Right, but how to pass it the arguments such as ignorecase or whatnot?

**Reply by LostInTheLogs** [reply]
that wasn't the problem martindavid was trying to solve, which is why I didn't include any information about arguments in my response (correct me if I'm wrong, maybe I misunderstood "full text search"), you can open a new question if you want that answered

**Reply by 1-800-jono** [reply]
The title of the discussion is literally: "Any example to configure telescope-live-grep-args.nvim plugin "

**Reply by LostInTheLogs** [reply]
the title seems incorrect (https://xyproblem.info/), it conflicts with the body of the question, it should be "How to get full text search capability" or something like this as this is what the author was trying to solve. they author didn't respond to my solution, so i don't know if I interpreted it correctly

**Reply by LostInTheLogs** [reply]
I don't want to solve the "how to configure telescope-live-grep-args.nvim" problem here, because I would ask a few questions depending on the full question, and since the original author is not responding the questions would be nested deep in the conversation and i have that aspect of github discussions, maybe I'm just being stubborn though, If my reasoning doesn't seem sound to you please tell me and I'll post the example here, but I'd prefer a new discussion that would be linked here

**Reply by 1-800-jono** [reply]
Fair enough. I started a new discussion here: https://github.com/LunarVim/LunarVim/discussions/4217
